### ISSUE TITLE
CBS News Live being blocked

### ISSUE THREAD

**Issue by yokoffing** [nextdns/metadata#574]
The NextDNS list currently blocks both `bidbarrel.cbsnews.com` and `tealium.cbsnews.com`. You have to allow both of these in order to stream `https://www.cbsnews.com/live/`.

**Comment by quantumpacket** [comment]
@yokoffing which list(s) in particular block these domains?

**Comment by yokoffing** [comment]
NextDNS Ads & Trackers Blocklist

@quantumpacket That is why I opened an issue here instead of elsewhere. It is occurring on the NextDNS list.

**Comment by scafroglia93** [comment]
There are other list with this domain inside

**Comment by quantumpacket** [comment]
The question is why were they blocked to begin with and in so many lists? Are they using the same domain to serve ads as well as live video? If so, I would rather keep it blocked so people are protected by default, and those who need the streaming can allow it if needed.

**Comment by pgl** [comment]
Just to chime in here because this thread was referenced elsewhere (https://github.com/StevenBlack/hosts/issues/1543), and I wanted to give a quick answer to @quantumpacket's question:

bidbarrel.cbsnews.com is used for real time bidding of ad placement.
tealium.cbsnews.com is used for Tealium, a tracking company.

**Comment by crssi** [comment]
The OP is half false.
Blocking `tealium.cbsnews.com` prevents the video to start.
Blocking `bidbarrel.cbsnews.com` does NOT BREAK the video stream.

Cheers

**Comment by yokoffing** [comment]
See [this thread](https://www.reddit.com/r/oisd_blocklist/comments/itfhom/oisd_domain_blocklist/gmucqcx?utm_source=share&utm_medium=web2x&context=3) for some testing I did browser side with the OISD domain list.

**Comment by crssi** [comment]
I did a through testing myself and it is as I said... you can safely block `bidbarrel.cbsnews.com` for stream to work.
You haven't mention `pubads.g.doubleclick.net` so I didn't too, but yes, it is true that blocking `pubads.g.doubleclick.net` also breaks the video.

But, as I said, blocking `bidbarrel.cbsnews.com` DOES NOT BREAK the video.

Cheers

**Comment by yokoffing** [comment]
> tealium.cbsnews.com is used for Tealium, a tracking company.

We found at Ghostery [two years ago?] that we had to allow this tracker to get this stream to work.

**Comment by yokoffing** [comment]
@crssi, @quantumpacket

 I was having some issues with my DNS that I recently resolved. This occur during the time I filed the report here.

The TL;DR: We can leave `bidbarrel.cbsnews.com` in the blocklist. Also, `tealium.cbsnews.com` is okay to be blocked by NextDNS (i.e. DNS-level) but must be allowed when using a browser content blocker (e.g. uBlock Origin or Ghostery).

My working theory is that the script(s) on `https://www.cbsnews.com/live/` need `tealium.cbsnews.com` in order to execute properly regardless of whether the `tealium` request connects successfully.
